src/pages/ProductList/index.jsx

```diff
@@ -0,0 +1,232 @@
+      setActiveFilters(newActiveFilters);
+    }
+  }
+  
+  return (
+    <Container collapsed={isMenuCollapse}>
+
+      <div className="container__sidebar">
+        <Sidebar>
+
+          <div className="sidebar__section" onClick={handleSidebar}>
+            <div className="sidebar__icon">
+              <img src={categoryIcon} alt="filter Icon"/>
+            </div>
+            <p>Categories</p>
+          </div>
+          <Divider />
+
+          <div className="sidebar__section">
+            <div className="sidebar__icon">
+              <img src={listIcon} alt="filter Icon"/>
+            </div>
+            <p>List</p>
+          </div>
+          { 
+            categoryList.map((category) => {
+              const id = category.id;
+              return (
+                <div key={id} className="sidebar__section">
+                  <div className="sidebar__icon check">
+                    <input 
+                      id={id} 
+                      type="checkbox" 
+                      onClick={(e) => handlerCheckbox(e,id)}
+                    />
+                  </div>
+                  <label htmlFor={id}>{category.data.name}</label>
+                </div>
+              );
+            })
+          }
+          <Divider />
+          
+          <div className="sidebar__section">
+            <div className="sidebar__icon">
+              <img src={activeIcon} alt="filter Icon"/>
+            </div>
+            <p>Actived</p>
+          </div>
+
+            { 
+              activeFilters.length > 0 && activeFilters.map((category) => {
+                const id = category.id;
+                return (
+                  <div key={id} className="sidebar__section">
+                    <div className="sidebar__icon check">
+                      <div className="sidebar__bullet"/>
+                    </div>
+                    <p>{category.data.name}</p>
+                  </div>
+                );
+              })
+            }
+
+        </Sidebar>
+      </div>
+
+      <div className="container__content">
+        
+        {
+          isLoading
+          ? <Spiner/>
+          : <>
+              {activeFilters.length === 0 
+              ? <h1>All products</h1>
+              : <h1>Filters activated</h1>
+              }
+
+              {
+                products.length === 0
+                ? <NoProducts> <h3>There are no products</h3> </NoProducts> 
+                : <ProductSection productsList={products} />
+              }
+            
+              <DividerSpace/>
+
+              <Pagination>
+                <nav>
+                  <a href="/productList" className="pagination__number">1</a>
+                  <a href="/productList" className="pagination__number">2</a>
+                  <a href="/productList" className="pagination__number">3</a>
+                  <a href="/productList" className="pagination__number">Next</a>
+                </nav>
+              </Pagination>
```

Comment:
You could separate some render stuff in several components, in order to make the code more reusable and shorter, for example, you can make  `<Pagination />` , `<Sidebar />,`